src/telliot_core/gas/legacy_gas.py

```diff
@@ -17,10 +22,18 @@ async def fetch_gas_price() -> int:
     return await ethgasstation("fast")
 
 
-async def ethgasstation(style: ethgastypes = "fast") -> int:
+async def ethgasstation(style: ethgastypes = "fast") -> Optional[int]:
     """Fetch gas price from ethgasstation in gwei"""
-    rsp = requests.get("https://ethgasstation.info/json/ethgasAPI.json")
-    prices = json.loads(rsp.content)
-    gas_price = int(prices[style])
-
-    return int(gas_price / 10)  # json output is gwei*10
+    i = 0
+    while i < 2:
```

Comment:
Maybe simplify the while loop to just `for _ in range(num_retries):` and add num_retries as an optional arg to this function with a default value of 2?